Easydict/Swift/View/AcknowListView/AcknowListView.swift

```diff
@@ -23,7 +23,10 @@ struct AcknowListView: View {
     var body: some View {
         NavigationStack {
             AcknowListSwiftUIView(acknowList: acknowList)
+                .listStyle(.sidebar)
         }
+        .hideWindowToolbarBackground()
```

Comment:
This will cause a UI conflict issue when we scroll the content.

So I guess we'd better not hide the toolbar background.

Instead use other ways (eg. containerBackground modifier) to achieve the consist effect of List and detail view.